src/rendering/renderers/shared/renderTarget/RenderTarget.ts

```diff
@@ -185,6 +193,14 @@ export class RenderTarget
 
     public destroy()
     {
-        throw new Error('Method not implemented.');
+        for (const colorTexture of this.colorTextures)
+        {
+            colorTexture.destroy();
+        }
```

Comment:
I think we also need to destroy the `depthTexture` as well if it exists

Reply:
Done, thanks!